core/hotspot/rule.go

```diff
@@ -64,51 +64,90 @@ func (t MetricType) String() string {
 // Rule represents the hotspot(frequent) parameter flow control rule
 type Rule struct {
 	// ID is the unique id
-	ID string `json:"id,omitempty"`
```

Comment:
The first letter of json cannot be changed. Compatibility issues need to be considered

Reply:
Sorry, I will pay attention to this wrong modification